notebooks/8-rotation-speeds-pt2/interm_statistics.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<p>No doubt you're already thinking about this, but:</p><ul><li>Before each function, we should introduce what we hope to achieve with it</li><li>Where we create a plot: what will this reveal about our data?</li><li>Perhaps at the top of the NB, or maybe before each exercise: explain what any stats measurements are.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>